infrastructure/helm.tf

```diff
@@ -0,0 +1,79 @@
+resource "helm_release" "load_balancer_controller" {
+  name             = "stefan-loadbalancer-controller"
+  repository       = "https://aws.github.io/eks-charts"
+  chart            = "aws-load-balancer-controller"
+  namespace        = "vegait-training"
+  create_namespace = true
+  set {
+    name  = "clusterName"
+    value = module.eks.cluster_name
+  }
+
+
+  set {
+    name  = "serviceAccount.name"
+    value = "vegait-load-balancer"
+  }
+
+  set {
+    name  = "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
+    value = module.iam_iam-github-oidc-provider.arn
+  }
+  set {
+    name  = "region"
+    value = var.region
+  }
+  set {
+    name  = "vpcId"
+    value = module.vpc.vpc_id
+  }
+  set {
+    name  = "defaultTargetType"
+    value = "ip"
+  }
+}
+
+
+resource "kubernetes_persistent_volume_claim" "postgres_pvc" {
```

Comment:
You don't define this in terraform as a separate resource, you should define storage class which will helm chart utilise to privison pvc inside kubernetes.